openedx/features/custom_fields/multiselect_with_other/helpers.py

```diff
@@ -1,3 +1,7 @@
+from openedx.features.custom_fields.multiselect_with_other.constants import \
+    OTHER_FIELD_CHECKBOX_VALUE
```

Comment:
Please run isort. Or directly alter the import statement to restrict it to a single line. Running isort restricts the import statements to a single line, as long as they are not exceeding the max length.